tests/test_launcher.py

```diff
@@ -706,3 +706,9 @@ def test_warning_in_linux():
             ui_mode=UIMode.NO_GUI_OR_GRAPHICS,
         )
         assert driver == FluentLinuxGraphicsDriver.NULL
+
+
+def test_no_warning_for_none_values(caplog):
+    driver = _get_graphics_driver(graphics_driver=None, ui_mode=None)  # noqa: F841
```

Comment:
Does this also work? Then don't need to add the flake disable directive
```suggestion
    _get_graphics_driver(graphics_driver=None, ui_mode=None) 
```